server/index-node/src/schema.graphql

```diff
@@ -7,6 +7,12 @@ scalar String
 type Query {
   indexingStatusesForSubgraphName(subgraphName: String!): [SubgraphIndexingStatus!]!
   indexingStatuses(subgraphs: [String!]): [SubgraphIndexingStatus!]!
+  proofOfIndexing(request: ProofOfIndexingInput!): Bytes
```

Comment:
In the interest of future-proofing, should this perhaps return an array of proof-of-indexing "slices"/"units"/"segments"... (whatever we call them)? Or would we combine them on the node before returning the result?

Reply:
This already combines them, which for The Network at least is already the desired outcome. If we want to output intermediate values for integration tests or something we could create a new API for that. I think we should leave that as a future improvement for now.